crates/napi/src/next_api/project.rs

```diff
@@ -1005,25 +1005,28 @@ pub fn project_entrypoints_subscribe(
         func,
         move || {
             async move {
-                let entrypoints_with_issues_op = get_entrypoints_with_issues_operation(container);
-                let EntrypointsWithIssues {
-                    entrypoints,
-                    issues,
-                    diagnostics,
-                    effects,
-                } = &*entrypoints_with_issues_op
-                    .read_strongly_consistent()
-                    .await?;
+                let entrypoints_with_issues_op = project_container_entrypoints_operation(container);
+                let entrypoints_operation = EntrypointsOperation::new(entrypoints_with_issues_op);
+                let (entrypoints, issues, diagnostics, effects) =
+                    strongly_consistent_catch_collectables(entrypoints_operation).await?;
```

Comment:
discussed in person

 strongly_consistent_catch_collectables needs to be run inside a turbotasks function not just here in the subscribe mapper
   - the reason is because it constructs a `cell` (im going to look into removing that, but don't block on that)